build-system/tasks/helpers.js

```diff
@@ -118,6 +122,19 @@ async function compileCoreRuntime(options) {
   await doBuildJs(jsBundles, 'amp.js', options);
 }
 
+/**
+ * Compiles the "core" utilies used by all bento extensions
+ *
+ * Outputs 2 scripts:
+ * 1) for direct consumption in the browser
+ * 2) for consumption by npm package users
+ * @param {!Object} options
+ * @return {Promise<void>}
+ */
+async function compileBentoRuntimes(options) {
```

Comment:
This is too similar imo, easy to typo (esp w/ VSCode auto-suggesting one or the other), maybe more verbose could be better here
```suggestion
async function compileBentoRuntimeAndCore(options) {
```

Reply:
updated